homework_18/Apple.java

```diff
@@ -0,0 +1,5 @@
+public class Apple extends Fruit {
+    public Apple() {
+        super(1.0F);
```

Comment:
Why do you put a float here, if class `Fruit` has a field `weight` with a type `double`?